Property_map/include/CGAL/property_map.h

```diff
@@ -503,6 +505,46 @@ make_boolean_property_map(Set& set_)
   return Boolean_property_map<Set>(set_);
 }
 
+/// \ingroup PkgProperty_map
+/// Read-write Property map turning a `CGAL::Point_3<K1>` into a 
+/// `CGAL::Point_3<K2>`, using a `Cartesian_converter`. 
+/// It means that `K1` and `K2` must be Cartesian Kernels.
+/// \cgalModels `ReadWritePropertyMap`
+template<class K1, class K2, class Vpm, class t_category = typename boost::property_traits<Vpm>::category>
+struct Kernel_converter_property_map
+{
+  typedef typename boost::property_traits<Vpm>::key_type key_type;
+  typedef typename K2::Point_3 value_type;
+  typedef typename K2::Point_3 reference;
+  typedef t_category category;
+  Vpm* vpm;
+  Kernel_converter_property_map(Vpm* vpm):vpm(vpm){}
+  Kernel_converter_property_map():vpm(NULL){}
+
+  friend value_type get(const Kernel_converter_property_map<K1, K2, Vpm, category>& pm, const key_type& k)
```

Comment:
Since the function is not a template one I think this will trigger a compilation error if the property map is not readable, even if get is not called. @lrineau do you agree?